openpype/hosts/houdini/plugins/create/create_composite.py

```diff
@@ -32,9 +32,17 @@ def create(self, subset_name, instance_data, pre_create_data):
             hou.text.expandString("$HIP/pyblish/"),
             "{}.$F4{}".format(subset_name, self.ext)
         )
+
+        frame_range = hou.playbar.frameRange()
+
         parms = {
             "trange": 1,
-            "copoutput": filepath
+            "copoutput": filepath,
+            # Unlike many other ROP nodes the composite node does not default
+            # to expression of $FSTART and $FEND so we preserve that behavior
+            # but do set the range to the frame range of the playbar
```

Comment:
> ..so we preserve that behavior but do set the range to the frame range of the playbar..

This seems to be a bit contradictory. We preserve it **but** do it differently?

How about:

> To match it more we at least default to the current playbar frame range
> on creation.

---

Or the question maybe is - if the others default to `$FSTART` and `$FEND` - why wouldn't we do that here too?

Reply:
tbh, I just stole these lines from here.
https://github.com/ynput/OpenPype/blob/3817ed150d1cd8a1fa890b673a8a046fdaf72f59/openpype/hosts/houdini/plugins/create/create_review.py#L45-L47

Reply:
Seems I should've had more coffee that day. ;)